tools/health/monitorUtil.py

```diff
@@ -46,8 +46,9 @@ def run(cmd, inData="") :
         p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stdin=subprocess.PIPE, stderr=subprocess.STDOUT)
         if (inData != ""):
             p.stdin.write(inData)
-        p.wait()
```

Comment:
I included one more change.
As per the python reference, it is recommended to use `communicate()` instead of `wait()` due to deadlock.
https://docs.python.org/2/library/subprocess.html#subprocess.Popen.wait

And the deadlock occurred on Cent OS